commands/init.go

```diff
@@ -151,3 +157,16 @@ func InitWebSocketClient() (*model.WebSocketClient, error) {
 	}
 	return client, nil
 }
+
+func InitUnixClient(socketPath string) (*model.Client4, error) {
+	tr := &http.Transport{
+		Dial: func(network, addr string) (net.Conn, error) {
+			return net.Dial("unix", socketPath)
+		},
+	}
+
+	client := model.NewAPIv4Client("http://_" + model.API_URL_SUFFIX)
```

Comment:
API suffix already gets added as part of the `NewAPIv4Client`, so this should be:

```suggestion
	client := model.NewAPIv4Client("http://_")
```